src/Raven.Server/Web/System/TcpConnectionInfoHandler.cs

```diff
@@ -77,6 +79,12 @@ public async Task GetRemoteTaskTcp()
             var remoteTask = GetStringQueryString("remote-task");
             var database = GetStringQueryString("database");
             var verifyDatabase = GetBoolValueQueryString("verify-database", false);
+
+            if (ServerStore.IsPassive())
+            {
+                throw new NodeIsPassiveException($"Can't fetch Tcp info from a passive node in url {this.HttpContext.Request.GetFullUrl()}");
```

Comment:
Actually why? Isn't it too strong requirement?

The commit message introducing this is "Handle passive nodes during subscription connection " although this will affect much more than just subscriptions, right? So want to double check that.

Reply:
This will affect only mechanisms that needs the TCP info for a database, which is only subscriptions AKAIK. 
We have a similar validation in our DocumentsHandler

Reply:
@karmeli87 What do you think? I see we have usage of this in `ReplicationLoader.GetPullReplicationTcpInfo`:

https://github.com/ravendb/ravendb/blob/ef7089368ffa23dd2d65a78bd6f2d155bcb53700/src/Raven.Server/Documents/Replication/ReplicationLoader.cs#L1048

Reply:
@karmeli87 Okay. Just to double check - do we need explicit handling for that in Pull Replication

@maximburyak Do we have a test for that scenario?

Reply:
@karmeli87 @maximburyak ping?

Reply:
this is fine